Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python
+FROM registry.access.redhat.com/ubi8/python-39:latest
 
 RUN pip install redis
```

Since you're at it, what about

```suggestion
RUN pip install redis==4.3.4
```